src/datumaro/components/exporter.py

```diff
@@ -339,10 +339,15 @@ def make_pcd_extra_image_filename(self, item, idx, image, *, name=None, subdir=N
         ) + self.find_image_ext(image)
 
     def make_video_filename(self, item, *, name=None):
-        if isinstance(item, DatasetItem) and isinstance(item.media, VideoFrame):
+        STR_WRONG_MEDIA_TYPE = "Video item's media type should be Video or VideoFrame"
```

Comment:
Very minor question: Can I ask you why you created a static variable for this message?